demo/app/lib/demo_platform_interface.dart

```diff
@@ -2,23 +2,19 @@ import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 
 import 'demo_method_channel.dart';
 
-abstract class HelloPlatform extends PlatformInterface {
+abstract class WalletPlatform extends PlatformInterface {
   /// Constructs a HelloPlatform.
```

Comment:
HelloPlatform? (I'm guessing this should say `WalletPlatform` now)

Reply:
yea it is walletplatform i think u commented on old file

Reply:
It says `HelloPlatform` in the comment

Reply:
> It says `HelloPlatform` in the comment

oh will fix it

Reply:
@talwinder50 I merged this PR - can you fix in next PR ?